Assets/Plugins/Editor/Uplift/Packages/PackageList.cs

```diff
@@ -80,16 +80,30 @@ public void RefreshPackages()
 
         public void LoadPackages(Repository repository)
         {
-            PackageRepo pr;
-            foreach (Upset package in repository.ListPackages())
+            using(LogAggregator la = LogAggregator.InUnity(
+                "Packages successfully loaded from {0}",
+                "Packages successfully loaded from {0}, but warning were raised",
+                "Error(s) occured while loading packages from {0}",
+                repository.ToString()
+            ))
             {
-                pr = new PackageRepo
+                PackageRepo pr;
+                try
                 {
-                    Package = package,
-                    Repository = repository
-                };
-                Packages.Add(pr);
-
+                    foreach (Upset package in repository.ListPackages())
+                    {
+                        pr = new PackageRepo
+                        {
+                            Package = package,
+                            Repository = repository
+                        };
+                        Packages.Add(pr);
+                    }
+                }
+                catch(Exception e)
+                {
```

Comment:
This failure breaks the whole listing. Is this what we want?
What are the conditions for this happening? This can only happen under ListPackages() right? If so maybe extract the try/catch just around it.